packages/web/src/components/audio-transaction-icon/AudioTransactionIcon.module.css

```diff
@@ -0,0 +1,64 @@
+.transactionIcon {
+  display: flex;
+  position: relative;
+  padding: 12px;
+}
+.transactionIconMini {
+  z-index: 1;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  position: absolute;
+  top: 6px;
+  left: 6px;
+  width: 18px;
+  height: 18px;
+  box-shadow: 0 0 0 2px var(--static-white);
```

Comment:
should this be `--static-white` or `--white`? I think it should blend in with the bg probably? (Julian Q)

Reply:
def a julian question, but I agree so will change for now